File: Array/Missing Number/Ques1.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n= nums.size();
        int totalNum= n*(n+1)/2;
        int NumsNumber=0;
        for (int i=0; i<n; i++ ){
         NumsNumber+=nums[i];
        }

        return totalNum-NumsNumber;
    }
};

File: Ques1.js

//Missing Number
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n= nums.length
   let  maxNum=n *(n+1)/2;
 let  numssub=0
    for (let i=0; i<n; i++){
        numssub+= nums[i]
    }

    return maxNum-numssub
};